CRITICAL - 2022-11-12 04:29:58 --> syntax error, unexpected ';', expecting ']'
#0 C:\xampp\htdocs\SIPM_polda\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\SIPM_polda\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
#3 [internal function]: spl_autoload_call('Config\\Filters')
#4 C:\xampp\htdocs\SIPM_polda\system\Config\Factories.php(136): class_exists('Config\\Filters')
#5 C:\xampp\htdocs\SIPM_polda\system\Config\Factories.php(106): CodeIgniter\Config\Factories::locateClass(Array, 'Filters')
#6 C:\xampp\htdocs\SIPM_polda\system\Common.php(198): CodeIgniter\Config\Factories::__callStatic('config', Array)
#7 C:\xampp\htdocs\SIPM_polda\system\Config\Services.php(258): config('Filters')
#8 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(248): CodeIgniter\Config\Services::filters(NULL, false)
#9 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('filters', Array)
#10 C:\xampp\htdocs\SIPM_polda\system\Config\Services.php(255): CodeIgniter\Config\BaseService::getSharedInstance('filters', NULL)
#11 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(248): CodeIgniter\Config\Services::filters()
#12 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(373): CodeIgniter\Config\BaseService::__callStatic('filters', Array)
#13 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\SIPM_polda\public\index.php(37): CodeIgniter\CodeIgniter->run()
#15 C:\xampp\htdocs\SIPM_polda\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2022-11-12 04:30:02 --> syntax error, unexpected ';', expecting ']'
#0 C:\xampp\htdocs\SIPM_polda\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\SIPM_polda\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
#3 [internal function]: spl_autoload_call('Config\\Filters')
#4 C:\xampp\htdocs\SIPM_polda\system\Config\Factories.php(136): class_exists('Config\\Filters')
#5 C:\xampp\htdocs\SIPM_polda\system\Config\Factories.php(106): CodeIgniter\Config\Factories::locateClass(Array, 'Filters')
#6 C:\xampp\htdocs\SIPM_polda\system\Common.php(198): CodeIgniter\Config\Factories::__callStatic('config', Array)
#7 C:\xampp\htdocs\SIPM_polda\system\Config\Services.php(258): config('Filters')
#8 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(248): CodeIgniter\Config\Services::filters(NULL, false)
#9 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('filters', Array)
#10 C:\xampp\htdocs\SIPM_polda\system\Config\Services.php(255): CodeIgniter\Config\BaseService::getSharedInstance('filters', NULL)
#11 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(248): CodeIgniter\Config\Services::filters()
#12 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(373): CodeIgniter\Config\BaseService::__callStatic('filters', Array)
#13 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\SIPM_polda\public\index.php(37): CodeIgniter\CodeIgniter->run()
#15 C:\xampp\htdocs\SIPM_polda\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2022-11-12 04:30:17 --> syntax error, unexpected ';', expecting ']'
#0 C:\xampp\htdocs\SIPM_polda\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\SIPM_polda\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
#3 [internal function]: spl_autoload_call('Config\\Filters')
#4 C:\xampp\htdocs\SIPM_polda\system\Config\Factories.php(136): class_exists('Config\\Filters')
#5 C:\xampp\htdocs\SIPM_polda\system\Config\Factories.php(106): CodeIgniter\Config\Factories::locateClass(Array, 'Filters')
#6 C:\xampp\htdocs\SIPM_polda\system\Common.php(198): CodeIgniter\Config\Factories::__callStatic('config', Array)
#7 C:\xampp\htdocs\SIPM_polda\system\Config\Services.php(258): config('Filters')
#8 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(248): CodeIgniter\Config\Services::filters(NULL, false)
#9 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('filters', Array)
#10 C:\xampp\htdocs\SIPM_polda\system\Config\Services.php(255): CodeIgniter\Config\BaseService::getSharedInstance('filters', NULL)
#11 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(248): CodeIgniter\Config\Services::filters()
#12 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(373): CodeIgniter\Config\BaseService::__callStatic('filters', Array)
#13 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\SIPM_polda\public\index.php(37): CodeIgniter\CodeIgniter->run()
#15 C:\xampp\htdocs\SIPM_polda\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#16 {main}
ERROR - 2022-11-12 04:35:18 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-11-12 04:35:18 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\SIPM_polda\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\SIPM_polda\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 C:\xampp\htdocs\SIPM_polda\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\SIPM_polda\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('hakim', 'username', Array, NULL)
#4 C:\xampp\htdocs\SIPM_polda\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('username', 'username', 'hakim', Array, Array)
#5 C:\xampp\htdocs\SIPM_polda\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 C:\xampp\htdocs\SIPM_polda\vendor\myth\auth\src\Controllers\AuthController.php(154): CodeIgniter\Controller->validate(Array)
#7 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(832): Myth\Auth\Controllers\AuthController->attemptRegister()
#8 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#9 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\SIPM_polda\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\SIPM_polda\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-12 04:37:40 --> Class name must be a valid object or a string
#0 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(248): Myth\Auth\Config\Services::activator(Object(Myth\Auth\Config\Auth), false)
#1 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#2 C:\xampp\htdocs\SIPM_polda\vendor\myth\auth\src\Config\Services.php(74): CodeIgniter\Config\BaseService::getSharedInstance('activator', NULL)
#3 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(248): Myth\Auth\Config\Services::activator()
#4 C:\xampp\htdocs\SIPM_polda\system\Common.php(929): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#5 C:\xampp\htdocs\SIPM_polda\vendor\myth\auth\src\Controllers\AuthController.php(184): service('activator')
#6 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(832): Myth\Auth\Controllers\AuthController->attemptRegister()
#7 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#8 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\SIPM_polda\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\SIPM_polda\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-11-12 04:38:21 --> Class name must be a valid object or a string
#0 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(248): Myth\Auth\Config\Services::activator(Object(Myth\Auth\Config\Auth), false)
#1 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#2 C:\xampp\htdocs\SIPM_polda\vendor\myth\auth\src\Config\Services.php(74): CodeIgniter\Config\BaseService::getSharedInstance('activator', NULL)
#3 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(248): Myth\Auth\Config\Services::activator()
#4 C:\xampp\htdocs\SIPM_polda\system\Common.php(929): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#5 C:\xampp\htdocs\SIPM_polda\vendor\myth\auth\src\Controllers\AuthController.php(184): service('activator')
#6 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(832): Myth\Auth\Controllers\AuthController->attemptRegister()
#7 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#8 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\SIPM_polda\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\SIPM_polda\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-11-12 04:41:47 --> Class name must be a valid object or a string
#0 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(248): Myth\Auth\Config\Services::activator(Object(Myth\Auth\Config\Auth), false)
#1 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#2 C:\xampp\htdocs\SIPM_polda\vendor\myth\auth\src\Config\Services.php(74): CodeIgniter\Config\BaseService::getSharedInstance('activator', NULL)
#3 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(248): Myth\Auth\Config\Services::activator()
#4 C:\xampp\htdocs\SIPM_polda\system\Common.php(929): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#5 C:\xampp\htdocs\SIPM_polda\vendor\myth\auth\src\Controllers\AuthController.php(184): service('activator')
#6 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(832): Myth\Auth\Controllers\AuthController->attemptRegister()
#7 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#8 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\SIPM_polda\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\SIPM_polda\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-11-12 04:44:00 --> Class name must be a valid object or a string
#0 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(248): Myth\Auth\Config\Services::activator(Object(Myth\Auth\Config\Auth), false)
#1 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#2 C:\xampp\htdocs\SIPM_polda\vendor\myth\auth\src\Config\Services.php(74): CodeIgniter\Config\BaseService::getSharedInstance('activator', NULL)
#3 C:\xampp\htdocs\SIPM_polda\system\Config\BaseService.php(248): Myth\Auth\Config\Services::activator()
#4 C:\xampp\htdocs\SIPM_polda\system\Common.php(929): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#5 C:\xampp\htdocs\SIPM_polda\vendor\myth\auth\src\Controllers\AuthController.php(184): service('activator')
#6 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(832): Myth\Auth\Controllers\AuthController->attemptRegister()
#7 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#8 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\SIPM_polda\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\SIPM_polda\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-11-12 04:48:49 --> authfilter filter must have a matching alias defined.
#0 C:\xampp\htdocs\SIPM_polda\system\Filters\Filters.php(493): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('authfilter')
#1 C:\xampp\htdocs\SIPM_polda\system\Filters\Filters.php(243): CodeIgniter\Filters\Filters->processAliasesToClass('before')
#2 C:\xampp\htdocs\SIPM_polda\system\Filters\Filters.php(163): CodeIgniter\Filters\Filters->initialize('/')
#3 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(393): CodeIgniter\Filters\Filters->run('/', 'before')
#4 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\SIPM_polda\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\SIPM_polda\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2022-11-12 04:51:28 --> You do not have sufficient permissions to access that page.
#0 C:\xampp\htdocs\SIPM_polda\system\Filters\Filters.php(173): Myth\Auth\Filters\RoleFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(393): CodeIgniter\Filters\Filters->run('admin', 'before')
#2 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\SIPM_polda\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\SIPM_polda\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-12 04:54:33 --> You do not have sufficient permissions to access that page.
#0 C:\xampp\htdocs\SIPM_polda\system\Filters\Filters.php(173): Myth\Auth\Filters\RoleFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(393): CodeIgniter\Filters\Filters->run('admin', 'before')
#2 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\SIPM_polda\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\SIPM_polda\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-12 04:55:37 --> You do not have sufficient permissions to access that page.
#0 C:\xampp\htdocs\SIPM_polda\system\Filters\Filters.php(173): Myth\Auth\Filters\RoleFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(393): CodeIgniter\Filters\Filters->run('admin', 'before')
#2 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\SIPM_polda\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\SIPM_polda\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-12 04:55:43 --> You do not have sufficient permissions to access that page.
#0 C:\xampp\htdocs\SIPM_polda\system\Filters\Filters.php(173): Myth\Auth\Filters\RoleFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(393): CodeIgniter\Filters\Filters->run('admin', 'before')
#2 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\SIPM_polda\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\SIPM_polda\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-12 05:02:52 --> You do not have sufficient permissions to access that page.
#0 C:\xampp\htdocs\SIPM_polda\system\Filters\Filters.php(173): Myth\Auth\Filters\RoleFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(393): CodeIgniter\Filters\Filters->run('admin', 'before')
#2 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\SIPM_polda\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\SIPM_polda\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-12 05:06:21 --> You do not have sufficient permissions to access that page.
#0 C:\xampp\htdocs\SIPM_polda\system\Filters\Filters.php(173): Myth\Auth\Filters\RoleFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(393): CodeIgniter\Filters\Filters->run('admin', 'before')
#2 C:\xampp\htdocs\SIPM_polda\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\SIPM_polda\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\SIPM_polda\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
